docs/README.md

```diff
@@ -1,8 +1,85 @@
-Documentation
-====
+# Website
 
-All documentation for Arcus Background Jobs which are published via GitHub Pages with Jekyll.
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-## Running Jekyll locally
+## Installation
 
-Running Jekyll locally to test your changes is super easy, learn more about it [here](https://jekyllrb.com/docs/#instructions).
\ No newline at end of file
+```console
+yarn install
+```
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Versioning
+https://docusaurus.io/docs/versioning
+
+You can use the version script to create a new documentation version based on the latest content in the docs directory. That specific set of documentation will then be preserved and accessible even as the documentation in the docs directory changes moving forward.
+
+### Tagging a new version
+
+1. First, make sure your content in the docs directory is ready to be frozen as a version. A version always should be based from master.
```

Comment:
Since we store everything in `/docs`, are you referring to `/docs` or `/docs/docs`?

Reply:
Renamed the root folder to `documentation` to avoid confusion

Reply:
As per https://github.com/arcus-azure/arcus.backgroundjobs/pull/73#issuecomment-893324833 let's please revert this because we follow the repo convention for this.

Let's just clarify in the docs by specifying in what folder the command needs to be run. Since the readme is in `docs` then I think we can just leave as is or mentioned ~"navigate to the root next to the readme"

Reply:
@tomkerkhove I renamed the `./docs/docs` folder to `./docs/next` or do you prefer `./docs/preview`?

Reply:
Renamed to `./docs/preview`